iOS/Flows/Conversation/_New/ConversationViewController.swift

```diff
@@ -232,32 +231,22 @@ extension ConversationViewController {
 // MARK: - SwipeableInputAccessoryViewDelegate
 
 extension ConversationViewController: SwipeableInputAccessoryViewDelegate {
-    
-    func swipeableInputAccessory(_ view: SwipeableInputAccessoryView, didConfirm sendable: Sendable) {
-        let alert = UIAlertController(title: "Send Method",
-                                      message: "Would you like to send your message?",
-                                      preferredStyle: .actionSheet)
 
-        if let currentIndexPath = self.collectionView.getCentermostVisibleIndex(),
-           let currentItem = self.dataSource.itemIdentifier(for: currentIndexPath) {
-            if case let .message(messageID) = currentItem {
-                let reply = UIAlertAction(title: "Reply", style: .default) { [unowned self] _ in
-                    Task {
-                        await self.reply(to: messageID, sendable: sendable)
-                    }
-                }
-                alert.addAction(reply)
-            }
+    func swipeableInputAccessory(_ view: SwipeableInputAccessoryView, didConfirm sendable: Sendable) {
```

Comment:
The SwipeableInputAccessoryViewDelegate now has two functions. One for sending new messages and one for replies.